std/process.d

```diff
@@ -1503,7 +1647,7 @@ Signal codes are defined in the $(D core.sys.posix.signal) module
 (which corresponds to the $(D signal.h) POSIX header).
 
 Throws:
-$(LREF ProcessException) on failure.
+$(LREF ProcessException) on failure or on attempt to wait for not owned process.
```

Comment:
Consider "detached" or "non-child" instead of "not owned", the "owned" term should be an implementation detail